packages/@react-spectrum/textfield/src/TextArea.tsx

```diff
@@ -25,22 +25,33 @@ function TextArea(props: SpectrumTextFieldProps, ref: RefObject<TextFieldRef>) {
     isReadOnly = false,
     isRequired = false,
     onChange,
+    defaultValue,
+    value,
     ...otherProps
   } = props;
 
   let inputRef = useRef<HTMLInputElement & HTMLTextAreaElement>();
+  let [count, check] = useState(0);
 
-  let onHeightChange = () => {
-    if (isQuiet) {
-      let input = inputRef.current;
-      input.style.height = 'auto';
-      input.style.height = `${input.scrollHeight}px`;
+  function changeDomHeight(dom: HTMLInputElement & HTMLTextAreaElement) {
```

Comment:
while this will probably work, it would be better not to have an extra piece of state just to make renders
what if we did something more along these lines?
```
let onHeightChange = useCallback(() => {
  if (isQuiet) {
    let input = inputRef.current;
    input.style.height = 'auto';
    input.style.height = `${input.scrollHeight}px`;
  }
}, [isQuiet, inputRef]);

useEffect(() => {
    if (inputRef.current) {
      onHeightChange();
    }    
}, [onHeightChange]);

...

onChange: chain(onChange, onHeightChange)
```

Reply:
At first I was thinking the `change dom height based on scrollheight` thing could be used later somewhere else across the project, so I created a new state `count` and let `useEffect` monitor `count`, change `count` state whenever change event is dispatched. Doing so the logic will be seperated.

But yes I agree with you, this way no extra state is needed, I will make a commit later.